mindsworth/Fluido-api
# #164849369 Configure eslint with the project

#### What does this PR do?
This PR is used to configure eslint with prettier in the project.

#### Description of Task to be completed?
- [x] add prettier configuration in eslint
- [x] add necessary packages
- [x] add editor configuration

#### How should this be manually tested?
- Run git fetch on develop
- Check out to the branch `ch-configure-eslint-164849369`
- Run `yarn install` to install all dependencies
- Run `yarn start:dev` to start the server
- In your text editor, you should lints whenever there is a syntax error.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#164849369](https://www.pivotaltracker.com/story/show/164849369)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
